lightclient-circuits/src/ssz_merkle.rs

```diff
@@ -94,10 +107,16 @@ pub fn verify_merkle_proof<F: Field, CircuitBuilder: CommonCircuitBuilder<F>>(
     Ok(())
 }
 
-pub const ZERO_HASHES: [[u8; 32]; 2] = [
-    [0; 32],
-    [
-        245, 165, 253, 66, 209, 106, 32, 48, 39, 152, 239, 110, 211, 9, 151, 155, 67, 0, 61, 35,
-        32, 217, 240, 232, 234, 152, 49, 169, 39, 89, 251, 75,
-    ],
-];
+lazy_static! {
```

Comment:
Why dont we just precompute like up to 64 levels then? Lolol

Reply:
cuz we know that we only need it for two levels as the size of the sync committee is always a power of 2 and the only other non-power of 2 collection is the block header (5 fields).